File: app/src/main/java/com/motrixi/datacollection/network/models/LanguageInfo.java
package com.motrixi.datacollection.network.models;

import java.io.Serializable;

/**
 * author : Jason
 * date   : 2020/12/8 10:50 AM
 * desc   :
 */
 public class LanguageInfo implements Serializable {

     public String code;
     public String language;


    @Override
    public String toString() {
        return "LanguageInfo{" +
                "code='" + code + '\'' +
                ", language='" + language + '\'' +
                '}';
    }
}

File: app/src/main/java/com/motrixi/datacollection/network/models/LanguageInfo.kt
package com.motrixi.datacollection.network.models

import java.io.Serializable

/**
 * @author jsqi
 * @time 2019/12/4 14:54
 */
class LanguageInfo: Serializable {

    var code: String? = null
    var language: String? = null

    constructor(code: String?, language: String?) {
        this.code = code
        this.language = language
    }


    override fun toString(): String {
        return "LanguageInfo(code=$code, language=$language)"
    }


}